server/graphql/v2/interface/IsMemberOf.js

```diff
@@ -128,15 +136,17 @@ export const IsMemberOfFields = {
         }
       }
 
-      if (!isNil(args.isApproved)) {
-        collectiveConditions.approvedAt = { [args.isApproved ? Op.not : Op.is]: null };
-      }
-      if (!isNil(args.isArchived)) {
-        collectiveConditions.deactivatedAt = { [args.isArchived ? Op.not : Op.is]: null };
+      const order = [
+        [sequelize.literal('"collective.memberCount"'), 'DESC'],
+        [sequelize.literal('"collective.totalAmountDonated"'), 'DESC'],
+      ];
```

Comment:
This should be an option of the `orderBy` argument. Or at least, we shouldn't fetch this heavy data when `orderBy` is referring to creation date.

Reply:
Should we use pseudo ordering arguments? (`memberCount` and `totalAmoundDonated`)

Reply:
Yes, it's not the easiest approach but I think that's what makes the most sense from a consumer point of view and gives flexibility for the future:

```gql
memberOf(orderBy: [MEMBER_COUNT, ASC])
```